src/script/ptl-tool.py

```diff
@@ -79,6 +79,22 @@
 # ...
 #
 #
+# Merging all PRs labeled 'wip-pdonnell-testing' into a new test branch but
+# NOT pushing that branch to ceph-ci repo:
+#
+# $ src/script/ptl-tool.py --pr-label wip-pdonnell-testing --branch main --no-build
```

Comment:
```suggestion
# $ src/script/ptl-tool.py --pr-label wip-pdonnell-testing --branch main --create-qa --no-push-ci
```

Reply:
It bears mentioning that we only push to ci if creating or updating a qa ticket.

Reply:
Okay, added a mention for this.